### ISSUE TITLE
Sass support

### ISSUE THREAD

**Issue by RPallas92** [thardy/generator-ngbp#31]
Is it possible to add Sass support, instead of Less?
Thank you

**Comment by thardy** [comment]
Absolutely.  It's not on my radar at the moment, but I'd welcome a pull request :)
